iocage_lib/iocage.py

```diff
@@ -90,8 +90,7 @@ def get_iocroot(self):
         Return:
                 string: with the iocroot name.
         """
-
-        return ioc_json.IOCJson(self.pool).json_get_value("iocroot")
+        return ioc_json.IOCJson().json_get_value("iocroot")
 
 
 class IOCage(object):
```

Comment:
Should we declare `self.zfs` from the zfs class we have created ?

Reply:
Actually, I think that would make more sense in another PR where we can consolidate zfs related functionality. Thoughts ?